virtual-organizations/GLOW.yaml

```diff
@@ -28,6 +28,10 @@ Contacts:
   VO Manager:
   - ID: 48e1c2f26dc3479a6cf9b2de7c79d654ac27b1d1
     Name: Miron Livny
+Credentials:
+  TokenIssuers:
+    - URL: https://scitokens.org/chtc
```

Comment:
Is this actually going to be the GLOW issuer?

Reply:
🤷🏻 It's what we used for StashCache below.